docs/server/settings.rst

```diff
@@ -250,6 +250,28 @@ Configuration settings for applications used by Pootle.
   - ``label`` specifying the text that will be displayed next to the mark.
 
 
+.. setting:: POOTLE_SCORING_CONF
```

Comment:
@unho, is this https://github.com/ta2-1/pootle/commit/edfff289d839fb1a1ad84beb66c7b665d3b1968e ok?